### PR TITLE
#268: [FlexView] remove confusing `auto` prop (closes #268)

### LINKED ISSUE #268: [FlexView] remove confusing `auto` prop

### PR DESCRIPTION
**Pull request by FrancescoCioria**
Issue #268

### CODE REVIEW

File: src/flex/FlexView.js [hunk 1]
```diff
@@ -76,6 +73,17 @@ const PropTypes = {
...
 export default class FlexView extends React.Component {
 
+  componentDidMount() {
+    this.logWarnings();
+  }
+
+  logWarnings = () => {
+    const { auto } = this.props;
+    if (!t.Nil.is(auto)) {
+      warn('"auto" is DEPRECATED. Use combination of "grow", "shrink" and "basis" instead');
```
Comment (giogonzo) [inline src/flex/FlexView.js]: we are not deprecating `auto` with this change, we are completely removing it :P

So this should be changed in something more strong, but probably could just be listed as breaking change in our CHANGELOG

Comment (FrancescoCioria) [inline src/flex/FlexView.js]: soo `"auto" is no longer supported. ...` ?

Comment (giogonzo) [inline src/flex/FlexView.js]: what I'm saying is it's not common to have warnings for features of old versions that are not there anymore (would be impossible to maintain such logs evolving a library..)

IMHO this should be explicit enough if we bump the semver and list the breaking changes somewhere, but also ccing @gabro here

Comment (gabro) [inline src/flex/FlexView.js]: semver + breaking change changelog :+1: 

provided someone is will to fix this in all our projects at once :P

Comment (FrancescoCioria) [inline src/flex/FlexView.js]: > provided someone is will to fix this in all our projects at once :P

volunteers? no one? ok... I'll do it... (you were referring to me anyway, weren't you? :( )
jokes apart, these changes should break only a very small portion of used `FlexView`s -> we should be able to fix them without too much effort

Comment (giogonzo) [inline src/flex/FlexView.js]: not necessarily at once, projects should fix their flexview version to what's more appropriate (i.e. sticking to the b-r-c one, not updated, for a while if that's better) ;)

**Pull request merged**